docs/asciidoc/big-graph-from-cypher.adoc

```diff
@@ -307,3 +310,185 @@ real    98m54.617s
 user    21m5.140s
 sys     37m35.852s
 ----
+
+=== UNWIND Batch Statements
+
+This mode use the optimizations explained in this https://medium.com/neo4j/5-tips-tricks-for-fast-batched-updates-of-graph-structures-with-neo4j-and-cypher-73c7f693c8cc[article]
+and leverages the `UNWIND` mode in order to speed-up the import process.
+
+This mode group:
+
+* nodes by the same labels
+* relationship by the same type and the same labels for start/end nodes in order to provide the `UNWIND` operation
+
+[source,cypher]
+----
+UNWIND [
+    {`UNIQUE IMPORT ID`: 0, properties: {`born`:date('2018-10-31'), `name`:"foo"}},
+    {`UNIQUE IMPORT ID`: 1, properties: {`born`:date('2017-09-29'), `name`:"foo2"}}
+] as row
+MERGE (n:`Foo`:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`: row.`UNIQUE IMPORT ID`})
+    SET n += row.properties;
+----
+
+[source,cypher]
+----
+UNWIND [{start: {`UNIQUE IMPORT ID`: 0}, end: {`name`: "bar"}, properties: {`since`:2016}}] as row
+MATCH (start:`Foo`:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`: row.start.`UNIQUE IMPORT ID`})
+MATCH (end:`Bar`{`name`: row.end.`name`})
+CREATE (start)-[r:`KNOWS`]->(end) SET r += row.properties;
+----
+
+==== UNWIND Batch Params
+
+In case of you choose CYPHER-SHELL as format you can also choose the UNWIND_BATCH_PARAMS optimization that enables the following
+so the generated file looks like the following
+[source,cypher]
+----
+:param rows => [{`UNIQUE IMPORT ID`: 0, properties: {`born`:date('2018-10-31'), `name`:"foo"}}, {`UNIQUE IMPORT ID`: 1, properties: {`born`:date('2017-09-29'), `name`:"foo2"}}]
+:begin
+UNWIND $rows AS row
+MERGE (n:`Foo`:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`: row.`UNIQUE IMPORT ID`}) SET n += row.properties;
```

Comment:
I think one issue here could be that it tries to MERGE on both Labels, we should always have only one Label in merge (the one from the primary key) and set the others with `SET n:Foo`